l10n_eu_oss/views/res_config_settings.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <record id="res_config_settings_view_form_l10n_eu_oss" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.l10n.eu.oss</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="account.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <div id="eu_oss" position="after">
```

Comment:
Not correct, eu_service is needed for the position.

Reply:
ups sorry, done.